peer.go

```diff
@@ -401,13 +413,19 @@ func (p *peer) QuitSignal() <-chan struct{} {
 
 // loadActiveChannels creates indexes within the peer for tracking all active
 // channels returned by the database.
-func (p *peer) loadActiveChannels(chans []*channeldb.OpenChannel) error {
+func (p *peer) loadActiveChannels(chans []*channeldb.OpenChannel) (
```

Comment:
`godoc` should be updated to detail the new return value

Reply:
Done